src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/automatic-remove/AutomaticRemoveView.tsx

```diff
@@ -0,0 +1,159 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import React, { ComponentProps, useState } from "react";
+import styles from "../dataBrokerProfiles.module.scss";
+import { Button } from "../../../../../../../../components/server/Button";
+import { useL10n } from "../../../../../../../../hooks/l10n";
+import getPremiumSubscriptionUrl from "../../../../../../../../functions/universal/getPremiumSubscriptionUrl";
+import { FixView } from "../../FixView";
+
+export type Props = Omit<ComponentProps<typeof FixView>, "children">;
+
+export function AutomaticRemoveView(props: Props) {
+  const l10n = useL10n();
+
+  const [selectedPlanIsYearly, setSelectedPlanIsYearly] = useState(true);
+
+  const dataBrokerCount = parseInt(
+    process.env.NEXT_PUBLIC_ONEREP_DATA_BROKER_COUNT as string,
+    10
+  );
+
+  return (
+    <FixView {...props}>
+      <div>
```

Comment:
All the content in here is the same as it was in `page.tsx`, but now wrapped in `<FixView>`, and with the data passed in as props.